sklearn/svm/_bounds.py

```diff
@@ -11,12 +11,7 @@
 
 def l1_min_c(X, y, *, loss="squared_hinge", fit_intercept=True, intercept_scaling=1.0):
     """
-    Return the lowest bound for C such that for C in (l1_min_C, infinity)
-    the model is guaranteed not to be empty. This applies to l1 penalized
-    classifiers, such as LinearSVC with penalty='l1' and
-    linear_model.LogisticRegression with penalty='l1'.
-
-    This value is valid if class_weight parameter in fit() is not set.
+    Return the lowest bound for C such that for C in (l1_min_C, infinity) the model is guaranteed not to be empty. This applies to l1 penalized classifiers, such as LinearSVC with penalty='l1' and linear_model.LogisticRegression with penalty='l1'. This value is valid if class_weight parameter in fit() is not set.
```

Comment:
Thanks @kshitijkapadni , any reason for changing the docstring to be one line? We generally like to keep lines <88 characters

Reply:
No Specific reason for it. I would correct that. Thanks.

Reply:
It just means that the short summary should be less than 88 characters.

You can then use a long summary. The following will solve the problem

```suggestion
    Return the lowest bound for C.

    The lower bound C is computed such that for C in (l1_min_C, infinity)
    the model is guaranteed not to be empty. This applies to l1 penalized
    classifiers, such as LinearSVC with penalty='l1' and
    linear_model.LogisticRegression with penalty='l1'.
```